lib/line/bot/event/things.rb

```diff
@@ -22,6 +22,11 @@ module ThingsType
         Unsupport = 'unsupport'
       end
 
+      # LINE Things API related events.
+      #
+      # https://developers.line.biz/en/reference/messaging-api/#device-link-event
+      # https://developers.line.biz/en/reference/messaging-api/#device-unlink-event
+      # https://developers.line.biz/en/reference/messaging-api/#scenario-result-event
```

Comment:
Is it may be better to use List?
![image](https://user-images.githubusercontent.com/9657/69933643-c102c400-1512-11ea-9329-554bf8f2c967.png)

example:
```
      #
      # * https://developers.line.biz/en/reference/messaging-api/#device-link-event
      # * https://developers.line.biz/en/reference/messaging-api/#device-unlink-event
      # * https://developers.line.biz/en/reference/messaging-api/#scenario-result-event
      # 
```